redirect.json

```diff
@@ -151,19 +157,21 @@
   {
     "redirect": "lovelace_dashboards",
     "name": "Lovelace: Dashboards",
+    "badge": "Lovelace Config",
     "description": "show your Lovelace dashboards",
```

Comment:
Also update this one to match?

Reply:
Show your lovelace dashboards configuration

Reply:
```suggestion
    "description": "show your Lovelace dashboard config",
```